src/popup/components/PayloadDetail.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
I suggest to rename the component to 'PayloadDetails' as "details" is not exacly what you mean here.